### PR TITLE
create to_json function 

### LINKED ISSUE #33: Add helper method for converting results to JSON
We need an easy way of turning the results from the `query` function into JSON.

### PR DESCRIPTION
**Pull request by NoorShamasneh**
Fixes #33 

add new function "to_json" inside new file "utilities.py"

### CODE REVIEW

File: railib/utilities.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+
```
Comment (larf311) [inline railib/utilities.py:1]: I believe this file needs the license as well.

File: railib/utilities.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    to_json converts Rel relations to dict object.
+    '''
+    
+    def nested_set(root, keys, value):
+        '''
+        set a value in a dict given a list of nested keys. 
+        '''
+        for i in range(len(keys) - 1):
+            key = keys[i]
+            nextValue = {} if type(keys[i+1]) is str else []
```
Comment (bradlo) [inline railib/utilities.py:22]: python does not use camelCase for identifiers

File: examples/show_json.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 RelationalAI, Inc.
```
Comment (bradlo) [inline examples/show_json.py:1]: wrong year in copyright header.

**Review by bradlo — CHANGES_REQUESTED** [review]

File: railib/utilities.py (hunk 2 above)
Comment (NoorShamasneh) [reply railib/utilities.py]: all variables now are snake_case 👍

File: examples/show_json.py (hunk 3 above)
Comment (NoorShamasneh) [reply examples/show_json.py]: fixed

File: railib/utilities.py (hunk 1 above)
Comment (NoorShamasneh) [reply railib/utilities.py:16]: added 👍

**Comment by larf311** [comment]
No, I think we were going to close this as I believe @bradlo had something else in mind.

**Pull request closed without merging**